heartbeat/beater/heartbeat.go

```diff
@@ -82,6 +84,10 @@ func (bt *Heartbeat) Run(b *beat.Beat) error {
 		return err
 	}
 
+	if b.ConfigManager.Enabled() {
+		bt.RunCentralMgmtMonitors(b)
```

Comment:
Hmmm, I just copied from https://github.com/elastic/beats/blob/master/metricbeat/beater/metricbeat.go#L109 but if it isn't necessary I'm glad to remove the conditional.

Is that safe?

Reply:
I checked how it's done in Metricbeat and it could have the same issue, I'm guessing that even in the improbable case of this happening, it would be fixed by the next pass on config apply. :+1: to leave it like this